doc/source/whatsnew/v0.16.0.txt

```diff
@@ -120,6 +121,44 @@ from a ``scipy.sparse.coo_matrix``:
    ss = SparseSeries.from_coo(A)
    ss
 
+.. _whatsnew_0160.enhancements.string:
+
+String Methods Enhancements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Following methods are accesible via ``.str`` accessor to apply the function to each values (:issue:`9282`, :issue:`9352`, :issue:`9386`, :issue:`9387`, :issue:`9439`)
```

Comment:
maybe also add that it is to make it more consistent with standard methods on strings